src/gui/widgets/LedCheckBox.cpp

```diff
@@ -107,9 +107,10 @@ void LedCheckBox::initUi( LedColor _color )
 {
 	setCheckable( true );
 
-	m_ledOnPixmap = new QPixmap( embed::getIconPixmap(
-					names[static_cast<std::size_t>(_color)].toUtf8().constData() ) );
-	m_ledOffPixmap = new QPixmap( embed::getIconPixmap( "led_off" ) );
+	static auto s_ledOnPixmap = embed::getIconPixmap(names[static_cast<std::size_t>(_color)].toUtf8().constData());
```

Comment:
This depends on the `_color` parameter, so it can't be `static`.

Both `Pixmap`s are still being deleted in the destructor even though they have static duration.

Reply:
Probably should just make them on the stack and not use `static` since an interal `QPixmapCache` is being used.